src/order/monotone/monovary.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2021 Yaël Dillies. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yaël Dillies
 -/
-import data.set.image
+import set_theory.ordinal.basic
```

Comment:
That file has been ported so I'm okay with changing this import. However, I think it is better to put your changes into a new file because it adds more material, based also on the discussions we had about PRs to modified files.